src/profile-logic/profile-store.js

```diff
@@ -4,6 +4,9 @@
 // @flow
 import { oneLine } from 'common-tags';
 
+const PUBLISHING_ENDPOINT = 'https://api.profiler.firefox.com/compressed-store';
```

Comment:
To test the code, you can change this url to `http://localhost:4243/compressed-store` and run profiler-server locally.

Note that because your local profiler-server dev environment doesn't point to the main store you won't be able to access the published profile. I think you could also change `getProfileUrlForHash` in `receive-profile.js` to point to the store's dev environment as well, but I haven't tried this yet.
Maybe we could make it easier to work on a dev environment, what do you think? Then maybe we could even have some integration tests...

Reply:
> Maybe we could make it easier to work on a dev environment, what do you think

Having an environment variable, and maybe a `yarn start-with-local-server` type of command would be good.

> Then maybe we could even have some integration tests...

We should discuss this further, but it seems like if we did this it would be good to have a mono-repo for that kind of test. I'm initially a bit nervous about having tests that run between repos.

Reply:
Maybe running the server with docker would make this easy even without a monorepo. Let's keep this in our mind :)